### PR TITLE
Final47 bugs list

### PR DESCRIPTION
**Pull request by kquinn1204**
@ahardin-rh I missed a few of the original final list I posted last week.

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -854,6 +854,16 @@ registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.8.0
 
 * Previously on some systems, the installer would communicate with Ironic before it was ready and fail. This is now prevented. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915295[*BZ#1915295*])
 
+* Previously the `cluster-baremetal-operator` used the incorrect logging library. This issue resulted in command line arguments not being consistent with other operators and not all k8s library logs were getting logged. Switching the logging library has fixed this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906143[*BZ#1906143*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:857]: s/operators/Operators
s/k8s/Kubernetes

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -854,6 +854,16 @@ registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.8.0
...
+
+* When using IPv6 on an interface, after a certain amount of time NetworkManager removes the link-local IPv6 address. This issue led to PXE boot failures occurring for nodes after the IPv6 link-local address is removed. A workaround has been added to toggle the interface IPv6 addr_gen_mode which will cause the link-local address to be added back. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1901040[*BZ#1901040*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:859]: s/NetworkManager/Network Manager

Comment (ahardin-rh) [reply release_notes/ocp-4-7-release-notes.adoc:859]: `addr_gen_mode`, which

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -876,6 +886,8 @@ This bug fix ensures Kubernetes is used as the source of truth. OVN-Kubernetes n
 
 * Previously when using dual stack deployments, worker node host names did not match the name inspected before deployment causing nodes to need manual approval. This is now fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1895909[*BZ#1895909*])
 
+* Bare metal provisioning now does not fail if there is a small (up to 1 hour) clock skew between the control plane and a host being deployed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906448[*BZ#1906448*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:889]: small, up to one hour, clock

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -924,6 +936,15 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
 
 * When an invalid Tuned profile is created, the `openshift-tuned` supervisor process may ignore future profile updates and fail to apply the updated profile. This bug fix keeps state information about Tuned profile application success or failure. Now `openshift-tuned` recovers from profile application failures on receiving new valid profiles. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1919970[*BZ#1919970*])
 
+*Performance Addon Operator*
+
+* Previously incorrect wait in the must-gather logic resulted in too early termination of log gathering. This issue resulted in depending on timing the log gathering operation being interrupted prematurely, leading to partial log collection.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:941]: Previously,

File: release_notes/ocp-4-7-release-notes.adoc [hunk 5]
```diff
@@ -924,6 +936,15 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
...
+This is now fixed by adding the correct wait in the must-gather logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906355[*BZ#1906355*])
+
+* Previously must-gather collected an unbounded amount of kubelet logs on all nodes. This issue resulted in an excessive amount of data being transferred and collected, with no clear benefit for the user.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:944]: Previously,

File: release_notes/ocp-4-7-release-notes.adoc [hunk 6]
```diff
@@ -924,6 +936,15 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
...
++
+This issue is fixed by collecting a bounded amount (last 8 hours) of kubelet logs only on worker nodes and not collecting kubelet logs on the control plane nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918691[*BZ#1918691*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:946]: amount, the last eight hours, of kubelet

File: release_notes/ocp-4-7-release-notes.adoc [hunk 7]
```diff
@@ -1170,6 +1191,12 @@ The root cause of this issue is unknown and no workaround currently exists.
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913279[*BZ#1913279*])
 
+* When attempting an IPI install on bare metal using the latest Dell firmware (04.40.00.00) nodes will not be deployed and an error will show in their status. This is due to Dell Firmware (4.40.00.00) using eHTML5 as the Virtual Console Plugin.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1194]: s/install/installation

Comment (ahardin-rh) [reply release_notes/ocp-4-7-release-notes.adoc:1194]: firmware (04.40.00.00), nodes

Comment (ahardin-rh) [reply release_notes/ocp-4-7-release-notes.adoc:1194]: s/Plugin/Plug-in

File: release_notes/ocp-4-7-release-notes.adoc [hunk 8]
```diff
@@ -1170,6 +1191,12 @@ The root cause of this issue is unknown and no workaround currently exists.
...
++
+To work around this issue, change the Virtual Console Plugin to HTML5 and run the deployment again. The Nodes should now be successfully deployed.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1196]: s/Plugin/Plug-in
s/Nodes/nodes

**Comment by kquinn1204** [comment]
@fromanirh Are you happy with the final text in the RN ? 
Bugs are: 
https://bugzilla.redhat.com/show_bug.cgi?id=1906355
https://bugzilla.redhat.com/show_bug.cgi?id=1918691

The description is in this section. 
https://deploy-preview-29575--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-bug-fixes

**Comment by fromanirh** [comment]
> @fromanirh Are you happy with the final text in the RN ?
> Bugs are:
> https://bugzilla.redhat.com/show_bug.cgi?id=1906355
> https://bugzilla.redhat.com/show_bug.cgi?id=1918691
> 
> The description is in this section.
> https://deploy-preview-29575--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-bug-fixes

the text looks good to me.

**Comment by kquinn1204** [comment]
@bfournie Are you happy with the final text in RN ?
The bug is 
https://bugzilla.redhat.com/show_bug.cgi?id=1901040

The description is in this section. 
https://deploy-preview-29575--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-bug-fixes

**Review by codyhoag — APPROVED** [review]

**Pull request merged**